### PR TITLE
Remove address from map when closing out search

### LINKED ISSUE #1152: The pinned address/ geography does not go away after the search is x'ed out
See bug in ADO for details:
https://dev.azure.com/NYCPlanning/ITD/_workitems/edit/5146

The screenshots show an older version of the PFF UI but this issue still exists and is reproducible.

### PR DESCRIPTION
**Pull request by bmarchena**
Closes #1152.

~~Also updates start:local-api script~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "start:local-api": "ember serve --environment=local-api",
+    "start:local-api": "ember serve --environment=devlocal",
```
Comment (bmarchena) [inline package.json:20]: @TylerMatteo I left this change in from when you helped me set up the local environment, thought it'd be helpful to update but if we should change it back I'm happy to do so

Comment (TylerMatteo) [reply package.json:20]: Go ahead and change it back. It's not ideal but `local-api` is helpful for when you _do_ want to use local instances of all APIs.

**Review by TylerMatteo — CHANGES_REQUESTED** [review]
Just undo the `package.json` change and then lgtm.

**Review by TylerMatteo — CHANGES_REQUESTED** [review]
Just undo the `package.json` change and then lgtm.

**Review by TylerMatteo — APPROVED** [review]

**Pull request merged**